datahub-frontend/play.gradle

```diff
@@ -14,9 +14,9 @@ configurations {
 
 dependencies {
   if (project.hasProperty('enableEmber') && project.getProperty('enableEmber').toBoolean()) {
-    assets project(path: ':datahub-web', configuration: 'assets')
+    // assets project(path: ':datahub-web', configuration: 'assets')
   } else {
-    assets project(path: ':datahub-web-react', configuration: 'assets')
+    // assets project(path: ':datahub-web-react', configuration: 'assets')
```

Comment:
nit: revert

Reply:
fyi I usually run `./gradlew datahub-frontend:build -x yarnBuild` so i get fast builds w/o having to worry about commenting / uncommenting this

Reply:
good call - done!